tests/app/submitter/test_converter.py

```diff
@@ -131,6 +131,28 @@ def test_display_address_should_be_set_in_payload_metadata(
     ]
 
 
+def test_instrument_id_is_not_in_output_payload_collection(
```

Comment:
The test name should highlight this is when the form type is not in the metadata. Previous tests use the words present/absent in test names.

Reply:
test_ref_period_end_date_is_not_in_output